task/s2i-java/0.1/s2i-java.yaml

```diff
@@ -237,15 +237,6 @@ spec:
       cat "$(workspaces.source.path)"/image-digest | tee $(results.IMAGE_DIGEST.path)
       echo -n "$(params.IMAGE)" | tee $(results.IMAGE_URL.path)
 
-      # Create and push sbom image
-      sbom_container=$(buildah from scratch)
-      sbom_container_mountpath=$(buildah mount $sbom_container)
-      cp sbom-cyclonedx.json sbom-purl.json $sbom_container_mountpath
```

Comment:
Does this mean we are getting rid of the sbom-purl.json format?

Reply:
that stays in the container image, but it's not pushed to the registry as separate image.

Reply:
ah, got it